PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,4 +7,4 @@ bug fixes why and what can be merged in a single item._
 
 ## How ?
 _It is optional but for complex PRs please provide information about the design,
-architecture, approach, etc.
+architecture, approach, etc._
```

Comment:
as I can see underscore here is used for Italic font (see begin of sentence)

Reply:
Yes, it seems that it (`_`) was missed.

Reply:
Ouch :) I missed it